data/faq20.toml

```diff
@@ -65,6 +65,11 @@ If you are using a Horizontal Pod Autoscaler (HPA) to scale on CPU and/or memory
 """
 type = "Best Practices"
 
+[[qna]]
+q = "Which least version of kubernetes is required for keda ?"
+a = "KEDA is designed, tested and is supported to be run on any [Kubernetes cluster](https://keda.sh/docs/operate/cluster/#requirements) that runs Kubernetes v1. 17.0 or above."
```

Comment:
This is unfortunately not correct; see https://keda.sh/docs/2.9/operate/cluster/#kubernetes-compatibility